### PR TITLE
Pry and rubocop

### PR DESCRIPTION
**Pull request by EdwardAndress**
- precursory to continuing development on this code
- enable use of binding.pry for debugging
- use Guard for Rubocop

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -528,7 +528,7 @@ Style/DefWithParentheses:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#method-parens'
   Enabled: false
 
-Style/DeprecatedHashMethods:
+Style/PreferredHashMethods:
```
Comment (misaka) [inline .rubocop.yml]: For anyone else wondering what happened here, `DeprecatedHashMethods` has been ... deprecated for `PreferredHashMethods`. 😄

**Comment by misaka** [comment]
👍 I did the same changes locally, but never cut the PR. 😛

File: .rubocop.yml (hunk 1 above)
Comment (EdwardAndress) [inline .rubocop.yml]: Thanks for the contribution :)

**Comment by danielatdigital** [comment]
👍

**Pull request merged**